dags/transportation/dockless/v_trips.sql

```diff
@@ -0,0 +1,21 @@
+CREATE MATERIALIZED VIEW v_trips AS 
+
+SELECT provider_id,
+	   provider_name, 
+	   device_id,
+	   vehicle_id,
+	   vehicle_type,
+	   propulsion_type,
+	   trip_duration, 
+	   trip_distance, 
+	   trip_distance * 0.000621371192 as trip_distance_miles,
```

Comment:
Some people may recognize the conversion from meters to miles, but not most. Maybe a variable here would help for clarity.

Reply:
hmmm good idea. I'll revise.

Reply:
Care to make this change? If not, we can move forwards and do it in a follow-up.